pyensembl/gene.py

```diff
@@ -73,6 +74,14 @@ def __eq__(self, other):
     def __hash__(self):
         return hash(self.id)
 
+    def __getstate__(self):
+        # Must be in order of __init__ arguments
+        return [self.id, self.name, self.contig, self.start, self.end, self.strand,
```

Comment:
Maybe we should just rename args to match the property? (I know that's a bigger change so maybe for a later PR where we'll also change serialization to use dicts)